src/applications/personalization/dashboard/reducers/folders.js

```diff
@@ -66,6 +70,17 @@ export default function folders(state = initialState, action) {
       );
     }
 
+    case FETCH_FOLDER_FAILURE: {
+      return set(
+        'data.currentItem',
+        {
+          errors: action?.errors,
```

Comment:
It might be nice to add `errors: null` on the initial state, just as an indicator that that property is or will be part of the state.

Reply:
Done 😄